### ISSUE TITLE
automatically cleanup other group_ldap_links

### ISSUE THREAD

**Issue by trissanen** [gitlabform/gitlabform#403]
Hi,

I'd like to be able to set group_ldap_links for a group definitely so that when I run gitlabform, it would also delete other previous ldap syncs for the project in a similar way like when you set enforce: true for members.

I haven't found a way to do this yet, not sure if such a flag already exists, but at least it's not documented

```
  mygroup/subgroup/*:
    group_ldap_links:
      enforce: true
      some_ldap_thing:
        provider: ldapmain
        cn: My_Group
        group_access: maintainer
```

**Comment by gdubicki** [comment]
Hi @trissanen! Thanks for asking about this.

No, we don't support this yet. We would have to implement it as mentioned here https://github.com/gitlabform/gitlabform/blob/ec60f49ebd2b94f8a7acce202429e240b814f893/gitlabform/processors/multiple_entities_processor.py#L59-L60.

As always, we are open to PRs with this feature. :)

**Comment by gdubicki** [comment]
Implemented in the PR above and release in [v3.2.0](https://github.com/gitlabform/gitlabform/releases/tag/v3.2.0) today. Please reopen if you have any issues with using this feature, @trissanen. :)
